fs/vfs/fs_uio.c

```diff
@@ -126,12 +126,14 @@ int uio_init(FAR struct uio *uio, FAR const struct iovec *iov, int iovcnt)
   ssize_t resid;
 
   memset(uio, 0, sizeof(*uio));
+
```

Comment:
unrelated whitespace changes

to me, all changes to this file looks cosmetic and unrelated to the rest of this commit.